sample-data/README.md

```diff
@@ -0,0 +1,4 @@
+# Sample data
+
+**#TODO: explain who recorded this data and for what purpose and what it's used for**
```

Comment:
It's a bunch of sensor data from test runs of the ventilator collected by @jlebar . I think each file has a documentation header inside itself.

Reply:
Ok, each one does explain the specifics of the experiment, true. But there should be a general note on how these are to be replayed, etc.. I will create a ticket for this.